### PR TITLE
demo: Refactoring

### PR DESCRIPTION
**Pull request by mlipok**
### Proposed changes

It makes easier to use wd_demo.au3 for checking if UDF works fine after each new changes, when messages from wd_demo.au3 differ form messages form UDF

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

Uncoherent messages in console

### What is the new behavior?

Coherent console messages, easier testing.
Removed magic number and added `$MB_TOPMOST`
Also fixed `_WD_Startup()` in relation to `If $_WD_DEBUG = $_WD_DEBUG_Info Then`

### Additional context

it is continuation for https://github.com/Danp2/au3WebDriver/pull/252

### System under test
Not Releated

### CODE REVIEW

File: wd_core.au3 [hunk 1]
```diff
@@ -1285,13 +1285,12 @@ Func _WD_Startup()
 
 		__WD_ConsoleWrite($sFuncName & ": OS:" & @TAB & @OSVersion & " " & @OSType & " " & @OSBuild & " " & @OSServicePack)
 		__WD_ConsoleWrite($sFuncName & ": AutoIt:" & @TAB & @AutoItVersion)
-		__WD_ConsoleWrite($sFuncName & ": au3WD UDF:" & @TAB & $__WDVERSION & $sUpdate)
+		__WD_ConsoleWrite($sFuncName & ": WebDriver UDF:" & @TAB & $__WDVERSION & $sUpdate)
```
Comment (Danp2) [inline wd_core.au3:1288]: This is already changed in my other PR

File: wd_core.au3 [hunk 2]
```diff
@@ -1285,13 +1285,12 @@ Func _WD_Startup()
...
 		__WD_ConsoleWrite($sFuncName & ": AutoIt:" & @TAB & @AutoItVersion)
-		__WD_ConsoleWrite($sFuncName & ": au3WD UDF:" & @TAB & $__WDVERSION & $sUpdate)
+		__WD_ConsoleWrite($sFuncName & ": WebDriver UDF:" & @TAB & $__WDVERSION & $sUpdate)
 		__WD_ConsoleWrite($sFuncName & ": WinHTTP:" & @TAB & $sWinHttpVer)
 		__WD_ConsoleWrite($sFuncName & ": Driver:" & @TAB & $_WD_DRIVER & $sDriverBitness)
 		__WD_ConsoleWrite($sFuncName & ": Params:" & @TAB & $_WD_DRIVER_PARAMS)
 		__WD_ConsoleWrite($sFuncName & ": Port:" & @TAB & $_WD_PORT)
-	Else
-		__WD_ConsoleWrite($sFuncName & ': ' & $sCommand)
+		__WD_ConsoleWrite($sFuncName & ": Command:" & $sCommand)
```
Comment (Danp2) [inline wd_core.au3:1293]: The original thought here is that `$_WD_DRIVER` and `$_WD_DRIVER_PARAMS` were already being shown above, so `$sCommand`, which is derived from these two variables, was only shown when $_WD_DEBUG <> $_WD_DEBUG_Info.

File: wd_core.au3 (hunk 1 above)
Comment (mlipok) [reply wd_core.au3:1288]: Yes. I know. I just want to keep it the same.

Comment (mlipok) [reply wd_core.au3:1288]: I revert it back and make speparate bigger PR.

File: wd_core.au3 (hunk 2 above)
Comment (mlipok) [reply wd_core.au3:1293]: I revert it back and make speparate bigger PR.

File: wd_core.au3 (hunk 1 above)
Comment (mlipok) [reply wd_core.au3]: As you can see wd_core.au3 is not changed in this PR anymore.

File: wd_core.au3 (hunk 2 above)
Comment (mlipok) [reply wd_core.au3]: As you can see wd_core.au3 is not changed in this PR anymore.

**Pull request merged**